blueprint/api.apib

```diff
@@ -2565,6 +2712,47 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(array[Stripe Plan Resource], required)
 + include JSON API Version
 
+## Stripe Subscription Attributes (object)
++ `cancelled-at`: null (string) - If the subscription has been canceled, the date of that cancellation.
++ created: `2016-07-08T03:03:51.967Z` (string)
++ `current-period-end`: `2016-07-08T03:03:51.967Z` (string) - End of the current period that the subscription has been invoiced for. At the end of this period, a new invoice will be created.
++ `current-period-start`: `2016-07-08T03:03:51.967Z` (string) - Start of the current period that the subscription has been invoiced for
++ `customer-id-from-stripe`: 'cus_9RnUvtsb41mD1g'  (string) - ID of the customer who owns the subscription
++ `ended-at`: null(string) - If the subscription has ended (either because it was canceled or because the customer was switched to a subscription to a new plan), the date the subscription ended
++ `id-from-stripe`: `sub_9RnVc9Cd03wV8X` (string, required)
++ `inserted-at`: `2016-07-08T03:03:51.967Z` (string)
++ `plan-id-from-stripe`: `monthly-donation-1` (string)
++ start: `2014-07-07T18:01:26.000Z` (string) - Date the most recent update to this subscription started.
++ status: `active` (enum[string])
+    + Members
+      + `trialing`
+      + `active`
+      + `past_due`
+      + `canceled`
+      + `unpaid`
```

Comment:
This is the order listed on the Stripe API, I'm thinking because of the order that the status is likely to have.  But could do Alpha instead.